template/android/build.gradle

```diff
@@ -6,14 +6,13 @@ buildscript {
         minSdkVersion = 16
         compileSdkVersion = 28
         targetSdkVersion = 28
-        supportLibVersion = "28.0.0"
     }
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:3.4.1")
+        classpath("com.android.tools.build:gradle:3.4.2")
```

Comment:
Actually the version should be `3.5.0` accordingly to this commit https://github.com/facebook/react-native/commit/4763000554c71be40b82a740651706781acad2e3#diff-d1933be33ff2cf0502f4e7420581a018

Reply:
They reverted it back according to this commit: https://github.com/facebook/react-native/commit/8750eb253858dd8b9d3480bd612b524fb38df1f1

Reply:
I didn't notice that, sorry